packages/@okta/vuepress-site/docs/reference/api/roles/index.md

```diff
@@ -942,15 +942,15 @@ A Binding represents an association of a principal, Role, and Resource Set that
 
 ### Create a new Binding
 
-<ApiOperation method="post" url="/api/v1/iam/resource-sets/${resourceSetId}/bindings" />
+<ApiOperation method="post" url="/api/v1/iam/resource-sets/${resourceSetIdOrLabel}/bindings" />
 
 Assigns a Custom Role by creating a Binding between the Role and the admin that targets an existing Resource Set
 
 #### Request parameters
 
 | Parameter      | Description                                                   | Param Type    | DataType       | Required |
 | :------------- | :------------------------------------------------------------ | :------------ | :------------- | :------- |
-| `resourceSetId`| ID of the Resource Set                                        | URL           | String         | TRUE     |
+| `resourceSetIdOrLabel`| Unique ID or label of the Resource Set                                       | URL           | String         | TRUE     |
```

Comment:
Can we make this ID as well?
We should standardize on using `ID` or `Unique ID`, and I see more uses of `ID` than `Unique ID`.